Tests/Recorded+Timeless.swift

```diff
@@ -9,14 +9,20 @@
 import RxTest
 import RxSwift
 
-func next<T>(_ value: T) -> Recorded<Event<T>> {
-    return Recorded(time: 0, value: .next(value))
+extension Recorded {
+    
+    public static func next<T>(_ element: T) -> Recorded<Event<T>> where Value == Event<T> {
```

Comment:
I don't think these extensions should be public for now.

Reply:
Oops forget to remove `public` keyword.

I've removed it now.

Thanks!